tests/components/Dropdown.test.tsx

```diff
@@ -1,11 +1,20 @@
+import React from 'react';
 import { render, screen } from '@testing-library/react';
-import userEvent from '@testing-library/user-event';
+import  {userEvent} from '@testing-library/user-event';
 import { Dropdown, DropdownProps } from '../../src/components/Dropdown/Dropdown';
 import { DropdownInput } from '../../src/components/Dropdown/DropdownInput';
 import { DropdownMenu } from '../../src/components/Dropdown/DropdownMenu';
 import { DropdownItem } from '../../src/components/Dropdown/DropdownItem';
+import { TextEncoder} from 'util';
+
+//due to react-dom not having TextEncoder, we need to mock it
+//before importing testSSR
+global.TextEncoder = TextEncoder;
```

Comment:
can we move this into `setup-env.ts`? that way we won't need to do this in multiple test files

Reply:
done